modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectActionLocalServiceImpl.java

```diff
@@ -402,6 +404,7 @@ public ObjectAction updateObjectAction(
 		objectAction.setActive(active);
 		objectAction.setConditionExpression(conditionExpression);
 		objectAction.setDescription(description);
+		_validateDescription(description);
```

Comment:
@Aquiiiles you should call this method along with the other validation methods, as I asked [here](https://github.com/liferay-bpm/liferay-portal/pull/4516#discussion_r2086979016)

```java
_validateActive(active, objectAction, objectDefinition);
_validateDescription(description);
_validateErrorMessage(errorMessageMap, objectActionTriggerKey);
```